jboss-eap-cd14-openshift/added/standalone-openshift.xml

```diff
@@ -437,7 +437,7 @@
                 </stack>
                 <stack name="tcp">
                     <transport type="TCP" socket-binding="jgroups-tcp">
-                        <property name="use_ip_addrs">true</property>
+                        <property name="use_ip_addrs">false</property>
```

Comment:
Just a note: for CD 15 this can be removed altogether as it is setting the value to its default. But for here (and the 7.2 variant) I think it's good to leave it. Makes this thing that's been the topic of discussion stand out a bit. :)

Reply:
Well, the PR for that is https://github.com/jboss-container-images/jboss-eap-modules/pull/42. I don't think we want to keep it.